dissect/target/target.py

```diff
@@ -730,7 +736,25 @@ def apply(self) -> None:
                 elif volume.is_encrypted(vol):
                     encrypted_volumes.append(vol)
                 else:
-                    self.open(vol)
+                    # We could be getting "regular" volume systems out of LVM or encrypted volumes
+                    # Try to open each volume as a regular volume system, or add as a filesystem if it fails
+                    try:
+                        vs = volume.open(vol)
+                    except Exception:
+                        # If opening a volume system fails, there's likely none, so open as a filesystem instead
+                        self.open(vol)
```

Comment:
I would recommend to add a log message here, for diagnostic purposes.

Reply:
Technically this is the happy path though (it's relatively rare that a volume has another default volume system), so this will result in a lot of unnecessary log messages. Would you still want a log message then?

Reply:
Should the code then not be restructured to make it reflect this fact better?

Reply:
Do you have a suggestion for doing so? Other than adding a comment.

Reply:
Did some research, I see that it is not possible currently.